src/Adapter/Module/CommandHandler/UpdateModuleStatusHandler.php

```diff
@@ -45,10 +45,7 @@ public function __construct(
 
     public function handle(UpdateModuleStatusCommand $command): void
     {
-        $module = $this->moduleRepository->getModule($command->getTechnicalName()->getValue());
-        if (!$module->disk->get('is_present')) {
-            throw new ModuleNotFoundException();
-        }
+        $this->moduleRepository->getPresentModule($command->getTechnicalName()->getValue());
```

Comment:
Perfect, this also should be added in the `GetModuleInfosHandler` so we have a uniformized way to get the module everywhere